File: sistemaacademicojcapi/app/Models/Persona.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;
use Illuminate\Database\Eloquent\Relations\HasOne;

class Persona extends Model
{
    protected $table = 'personas';
    protected $primaryKey = 'id_persona';
    public $timestamps = false;

    protected $fillable = [
        'nombres_persona',
        'apellidos_pat',
        'apellidos_mat',
        'sexo_persona',
        'direccion_persona',
        'nacionalidad_persona',
        'fecha_nacimiento',
        'celular_persona',
        'fotografia_persona',
    ];

    // Relaciones

    public function persona_roles(): HasMany
    {
        return $this->hasMany(PersonaRol::class, 'personas_id_persona', 'id_persona');
    }

    public function documento(): HasOne
    {
        return $this->hasOne(Documento::class, 'personas_id_persona', 'id_persona');
    }
}

File: sisitemacademicoappmovil/SistemaAcademicoJCT/app/src/main/java/com/jsq/sistemaacademicojct/model/Persona.java
package com.jsq.sistemaacademicojct.model;

public class Persona {
    private int id_persona;
    private String nombres_persona;
    private String apellidos_pat;
    private String apellidos_mat;
    private String sexo_persona;
    private String fecha_nacimiento;
    private String direccion_persona;
    private String nacionalidad_persona;
    private String celular_persona;
    private String fotografia_persona;

    public String getNombres_persona() {
        return nombres_persona;
    }

    public String getApellidos_pat() {
        return apellidos_pat;
    }

    public String getApellidos_mat() {
        return apellidos_mat;
    }


    // Puedes agregar más getters si los necesitas
}
